3.7/debian-10/rootfs/opt/bitnami/scripts/zookeeper-env.sh

```diff
@@ -142,4 +144,8 @@ export ZOO_SERVER_USERS="${ZOO_SERVER_USERS:-}"
 export ZOO_CLIENT_PASSWORD="${ZOO_CLIENT_PASSWORD:-}"
 export ZOO_SERVER_PASSWORDS="${ZOO_SERVER_PASSWORDS:-}"
 
+# Admin server https://zookeeper.apache.org/doc/r3.5.7/zookeeperAdmin.html#sc_adminserver
+export ZOO_ADMIN_SERVER_ENABLE="${ZOO_ADMIN_SERVER_ENABLE:-true}"
```

Comment:
For consistency with other similar environment variables, could you rename it to `ZOO_ENABLE_ADMIN_SERVER`? Also, please change it's default value to `yes`.

```suggestion
export ZOO_ENABLE_ADMIN_SERVER="${ZOO_ENABLE_ADMIN_SERVER:-yes}"
```

Reply:
Hey! 
The variable has been renamed. 
Regarding the default value, it seems the `true` is correct.
According to the spec, values have to be true/false. https://zookeeper.apache.org/doc/r3.5.7/zookeeperAdmin.html#sc_adminserver_config
The function `is_boolean_yes` are well recognized `yes` and `true` as well.

Reply:
I see. In that case we should change it, because we try to make all `ENABLE` environment variables support yes/no as values.

In order to support that, you may need to make these changes in `_initialize`:

```
-zookeeper_conf_set "$ZOO_CONF_FILE" admin.enableServer "$ZOO_ENABLE_ADMIN_SERVER"
+zookeeper_conf_set "$ZOO_CONF_FILE" admin.enableServer "$(is_boolean_yes "$ZOO_ENABLE_ADMIN_SERVER" && echo "true" || echo "false")"
```

Reply:
it's done. readme updated as well